sonic_platform_base/module_base.py

```diff
@@ -262,28 +262,22 @@ def get_state_info(self):
         """
         raise NotImplementedError
 
-    def get_bus_info(self, module_name):
+    def get_bus_info(self):
         """
-        Retrieves the bus information for the specified by "module_name" on a SmartSwitch.
+        Retrieves the bus information. Returns None for non-smartswitch chassis.
```

Comment:
@vvolam do we need to mention smartswitch or not here? This is abstract class and let the implementation choose to decide whether its supported or not. The calller should validate the return value

Reply:
@prgeor Fixed it. Thank you!